config/locales/candidate_interface/gcse_details.yml

```diff
@@ -30,6 +31,11 @@ en:
         You need at least a grade C.
 
         If you do not have this, contact teacher training providers to discuss your eligibility for their courses. You might be able to retake an exam or do an equivalence test.
+      international_baccalaureate_middle_years_programme: |
+        You normally need to have received at least 3 points for the subject in your International Baccalaureate Middle Year Programme, in order to demonstrate equivalence to a GCSE 4 (C) or above.
```

Comment:
what does 'for the subject' mean here?

Reply:
My understanding comes from this page, although I'm not totally clear: https://www.ibo.org/programmes/middle-years-programme/assessment-and-exams/grading-and-awards/